manager/cve_handler.py

```diff
@@ -152,6 +164,129 @@ def _full_query(rh_account_id, synopsis):
                 .where(system_is_active(rh_account_id=rh_account_id))
                 .where(system_is_vulnerable(rule_subselect=False)))
 
+    @staticmethod
+    def _unpatched_full_query(rh_account_id, synopsis):
+        # pylint: disable=singleton-comparison
+        unfixed_subq = (SystemVulnerablePackage
+                        .select(
+                                CveMetadata.id.alias("cve_id"),
+                                CveMetadata.cve,
+                                SystemPlatform.id.alias("system_id"),
+                                SystemPlatform.inventory_id,
+                                SystemPlatform.display_name,
+                                SystemPlatform.last_evaluation,
+                                SystemPlatform.advisor_evaluated,
+                                SystemPlatform.last_upload,
+                                SystemPlatform.stale_timestamp,
+                                SystemPlatform.stale_warning_timestamp,
+                                SystemPlatform.culled_timestamp,
+                                Value(None).alias("rule_id"),
+                                Value(False).alias("rule_active"),
+                                Value(None).alias("description_text"),
+                                Value(None).alias("reason_text"),
+                                Value(None).alias("resolution_text"),
+                                Value(0).alias("kbase_node_id"),
+                                Value(None).alias("more_info_text"),
+                                Value(None).alias("status_text"),
+                                Value(None).alias("rule_hit_details"),
+                                Value(datetime.min).alias("when_mitigated"),
+                                SystemVulnerablePackage.first_reported,
+                                Value(None).alias("mitigation_reason"),
+                                Value(None).alias("advisories"),
+                                Value(0).alias("status_id"),
+                                Value(False).alias("advisory_available"),
+                                Value(remediation.MANUAL.value).alias("remediation_type_id"),
+                                )
+                        .distinct()
+                        .join(SystemPlatform, on=(SystemVulnerablePackage.system_id == SystemPlatform.id) &
+                                                        system_is_active(rh_account_id=rh_account_id))
+                        .join(VulnerablePackageCVE, on=(SystemVulnerablePackage.vulnerable_package_id == VulnerablePackageCVE.vulnerable_package_id))
+                        .join(CveMetadata, on=(VulnerablePackageCVE.cve_id == CveMetadata.id))
+                        .where(SystemVulnerablePackage.rh_account_id == rh_account_id)
+                        )
+
+        fixed_subq = (SystemVulnerabilities
```

Comment:
this should re-use the query from `_full_query` and not repeat it

Reply:
I don't get it yet, but if I were to learn from mistakes of the past, I should know, there is just something yet for me to see :smile:

Reply:
I will think about it :slightly_smiling_face: